uaclient/api/u/unattended_upgrades/status/v1.py

```diff
@@ -224,10 +224,7 @@ def _status(cfg: UAConfig) -> UnattendedUpgradesStatusResult:
     unattended_upgrades_last_run = _get_unattended_upgrades_last_run()
 
     unattended_upgrades_cfg = get_apt_config_values(
-        set(
```

Comment:
That's indeed an issue and you are right that we should not consider all config options set on `unatteded-upgrade`  to see if it is enabled or not.

However, one of the API fields we have is the `raw_config` one. On that field, we should capture all of the configs set for `unattended-upgrade`. By removing this extra line here, we loose visibility of `unattended-upgrade` config set through apt in `raw_config`

What if we instead change the `_is_unattended_upgrades_running` to only check the keys `UNATTENDED_UPGRADES_CONFIG_KEYS` ? In that sense, we would not affect the `raw_config` and we will also fix the issue that you found in the API